lms/static/scripts/frontend_apps/services/client-rpc.js

```diff
@@ -1,11 +1,46 @@
 import { Server, call as rpcCall } from '../../postmessage_json_rpc';
+import { apiCall } from '../utils/api';
 
 /**
  * @typedef User
  * @prop {string} displayName
  * @prop {string} userid
  */
 
+/**
+ * @typedef ServiceConfig
+ * @prop {string} grantToken
+ */
+
+/**
+ * The subset of the Hypothesis client configuration that `ClientRpc` references.
+ *
+ * The backend will set other configuration which is just forwarded to the client
+ * and not touched by `ClientRpc`.
+ *
+ * See https://h.readthedocs.io/projects/client/en/latest/publishers/config/.
+ *
+ * @typedef ClientConfig
+ * @prop {[ServiceConfig]} services
+ */
+
+/**
+ * Return the time at which a JWT token expires.
```

Comment:
Orthogonal question: What's our current convention on documenting functions that can `throw`?